db/migrate/20211202183101_estimated_population.rb

```diff
@@ -0,0 +1,12 @@
+class EstimatedPopulation < ActiveRecord::Migration[5.2]
+  def change
+    create_table :estimated_populations, id: :uuid do |t|
+      t.belongs_to :region, index: {unique: true}, foreign_key: true, type: :uuid, null: false
+      t.integer :population, null: false
+      t.string :diagnosis, null: false, default: "HTN"
+      t.uuid :created_by
+      t.uuid :updated_by
```

Comment:
```suggestion
      t.timestamps, null: false
```

You'll need to rerun this migration and sync up chnages to structure.sql for this.